modules/proposals/codex/src/proposal_types/text_proposal.rs

```diff
@@ -0,0 +1,38 @@
+use codec::{Decode, Encode};
+use rstd::prelude::*;
+
+use rstd::str::from_utf8;
+use srml_support::{dispatch, print};
+
+use crate::{ProposalExecutable, ProposalType};
+
+/// Text (signal) proposal executable code wrapper. Prints its content on execution.
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, Default)]
+pub struct TextProposalExecutable {
+    /// Text proposal title
+    pub title: Vec<u8>,
+
+    /// Text proposal body (description)
+    pub body: Vec<u8>,
```

Comment:
Names `body` and `text` look confusing to me. Just saying.

Reply:
Will rename to the "description"!